src/main/java/com/github/manolo8/darkbot/core/manager/StarManager.java

```diff
@@ -138,7 +138,9 @@ private StarManager() {
                 .addGG(453, "GoP Easy 4", "GoP E").accessOnlyBy(235, "GoP Easy 3")
                 .addGG(454, "GoP Easy 5", "GoP E").accessOnlyBy(235, "GoP Easy 4")
                 .addGG(455, "GoP Easy Final", "GoP E").accessOnlyBy(235, "GoP Easy 5").exitBy(1)
-                .addGG(470, "Plutus' Trove of Riches Easy", "PToR E").accessOnlyBy(235, "GoP Easy Final").exitBy(1);
+                .addGG(470, "Plutus' Trove of Riches Easy", "PToR E").accessOnlyBy(235, "GoP Easy Final").exitBy(1)
+                .addGG(471, "Treacherous Domain Easy", "TD E").accessOnlyBy(238, HOME_MAPS)
```

Comment:
do these maps not have an exit portal? is it an automatic tp-out?

Reply:
does not have exit portal, just kicks them out of map once its completed